### PR TITLE
feat: bring yaml

### PR DESCRIPTION
**Pull request by eladcon**

### CODE REVIEW

File: yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+let file = fs.readFile("./test.yml");
+let value = yaml.parseValue(file);
+yaml.stringifyValue(json);
+
+new cloud.Function(inflight () => {
+  let value = yaml.parse(file);
+  yaml.stringify(json);
+});
+```
```
Comment (Chriscbr) [inline yaml/README.md:30]: We should totally include this in the built-in standard library!

Comment (eladb) [reply yaml/README.md:30]: Don't we already have some yaml stuff in the standard library?

Comment (eladb) [reply yaml/README.md:30]: I agree this should be part of the stdlib.

File: yaml/lib.w [hunk 2]
```diff
@@ -0,0 +1,33 @@
+pub struct ParseOptions {
+  strict: bool?;
+}
+
+pub struct StringifyOptions {
+  indent: num?;
+}
+
+pub class Util {
```
Comment (eladb) [inline yaml/lib.w:9]: what's the benefit of this duplication? Why not just make the extern methods public?

**Review by eladb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**